ocs_ci/ocs/debug.py

```diff
@@ -0,0 +1,313 @@
+import logging
```

Comment:
Please change the file name to another suitable name, the current name is too generic